Core.ApplicationServices/UserService.cs

```diff
@@ -280,5 +289,48 @@ public void Dispose()
                     )
                 );
         }
+
+        public Maybe<OperationError> DeleteUserFromKitos(Guid userUuid)
+        {
+            using var transaction = _transactionManager.Begin();
+
+            var user = _userRepository.AsQueryable().ByUuid(userUuid);
+            if (user == null)
+                return new OperationError(OperationFailure.NotFound);
+            if(_activeUserIdContext.GetValueOrDefault().ActiveUserId == user.Id)
```

Comment:
incorrect use
default may be null
To use the maybe correctly, do as:
`_activeUserIdContext.Select(context=>context.ActiveUserId == user.Id).GetValueOrDefault()`
That will return a boolean